lib/Minify/HTTP/Encoder.php

```diff
@@ -168,7 +168,7 @@ public function sendHeaders()
 	public function sendAll()
 	{
 		$this->sendHeaders();
-		echo $this->_content;
+		echo esc_html( $this->_content );
```

Comment:
I'm not sure if this needs to get escaped.

Reply:
Switched to wp_kses with call to Util_Ui::get_allowed_html_for_wp_kses_from_content for allowed HTML tags